elki/src/main/java/de/lmu/ifi/dbs/elki/evaluation/clustering/PairCounting.java

```diff
@@ -138,8 +138,7 @@ protected PairCounting(ClusterContingencyTable table) {
    */
   public double fMeasure(double beta) {
     final double beta2 = beta * beta;
-    double fmeasure = ((1 + beta2) * pairconfuse[0]) / ((1 + beta2) * pairconfuse[0] + beta2 * pairconfuse[1] + pairconfuse[2]);
```

Comment:
IMHO it makes sense to keep a named variable here, to emphasize the result of this equation is called the f-measure.